components/modal.jsx

```diff
@@ -1,10 +1,9 @@
-var React = require('react');
-var ReactAddons = require('react/addons');
+var React = require('react/addons');
```

Comment:
Yep, it's the equivalent of including their JS bundle that includes all the mixins. The original code that was committed was actually in error--React doesn't complain when a mixin object is `undefined`, which it was, and I'm just fixing things here.